File: dayEight/python/double_even_add_one_to_odd.py
def isEven(number):
    isEven = False
    if(number % 2 == 0):
        isEven = True;
    
    return isEven;


def doubleEvenAddOneToOdd(arrayOfNumbers):
    changedArray = []
    
    for index in range(len(arrayOfNumbers)):
        if(isEven(arrayOfNumbers[index])):
            changedArray.append(arrayOfNumbers[index] * 2)
        else:
            changedArray.append(arrayOfNumbers[index] + 1)
        
    return changedArray;

File: dayEight/Javascript/DoubleEvenAddOneToOdd.js
let isEven(number){
    isEven = false;
    if(number % 2 == 0){
        isEven = true;
    }
    
    return isEven;
}

let doubleEvenAddOneToOdd(arrayOfNumbers){
    let changedArray = [];
    
    for(int index = 0; index < arrayOfNumbers.length; index++){
        if(isEven(arrayOfNumbers[index])){
            changedArray.push(arrayOfNumbers[index] * 2);
        }else{
            changedArray.push(arrayOfNumbers[index] + 1);
        }
    }
    
    return changedArray;
}
